### DISCUSSION TITLE
Adjustable Row Height

### DISCUSSION THREAD

**Discussion by danieldonorbox** [community/community#6962] [Projects and Issues]
It would be nice if Github Tables would allow the user to adjust the height of a row. The use-cases would include allowing the user to see all text in either the following circumstances:

- a list of items/text inside a given cell
- or, a large body of text inside a given cell.

Thanks!

**Comment by leighjboyd** [comment]
I agree.  I have a title field, but also want a description field, so I can quickly scan titles, but read more deeply the description before i click on the issue and open up an new tab (many many new tabs get created quickly) 
So when I added the description text, it only allows one row, and if its longer than that, you cant read it all!  No height adjustment, no dropdown to make cell larger, etc.
Also, when I open the issue, the description text is not visible, since I guess it is just a user defined field, so issue doesn't care about it.  
So I prefer if another type of field, called Description, would be added more permanently as well

**Comment by regfunkid** [comment]
I found a solution to not having directly adjustable table rows!
You can insert a `<br>` in a table to force it to make more room, without creating another table row.
```
Column One|Column Two
-|-|
Info1|Example Text
Info2|Example Text<br>More Example Text<ul><li>Example 1</li><li>Example 2</li><li>[Example 3 With a Link](https://github.com)</li></ui>
```
Column One|Column Two
-|-|
Info1|Example Text
Info2|Example Text<br>More Example Text<ul><li>Example 1</li><li>Example 2</li><li>[Example 3 With a Link](https://github.com)</li></ui>
